packages/web/src/common/store/cache/collections/sagas.js

```diff
@@ -517,8 +557,8 @@ function* confirmAddTrackToPlaylist(userId, playlistId, trackId, count) {
       undefined,
       {
         operationId: PlaylistOperations.ADD_TRACK,
-        parallelizable: true,
-        useOnlyLastSuccessCall: true
```

Comment:
Should add this back for the non entity manager case:
`useOnlyLastSuccessCall: !playlistEntityManagerEnabled`